modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/model.scala

```diff
@@ -222,7 +257,12 @@ object Queue {
 /**
   * A list of Steps grouped by target and instrument.
   */
-case class Sequence[A](id: String, steps: NonEmptyList[Step[A]])
+case class Sequence[A](
+  id: String,
+  steps: NonEmptyList[Step[A]],
+  // Unrooted executions
```

Comment:
Are you using `Zipper` from scalaz? So we'd have `focus`, `lefts` and `rights`. I somehow thought that focus would be in either `lefts` or `rights` but I guess being separated is possible too.

What would be the type in the zipper? at the level of `Step`?